packages/php/src/Metrics.php

```diff
@@ -196,7 +196,7 @@ public function constructPayload(string $log_id, Request $request, Response $res
                     'entries' => [
                         [
                             'pageref' => $request->url(),
-                            'startedDateTime' => date('c', (int) $request_start),
+                            'startedDateTime' => date('Y-m-d\TH:i:sp', (int) $request_start),
```

Comment:
What are the implications of this change?

Reply:
`startedDateTime` used to look like `2022-08-18T18:25:57+00:00` whereas now it'll be `2022-08-18T18:26:13Z` to match how our other SDKs handle it.